PurchasesTests/Purchasing/PurchasesTests.swift

```diff
@@ -2016,6 +2016,89 @@ class PurchasesTests: XCTestCase {
         verifyUpdatedCaches(newAppUserID: identityManager.currentAppUserID)
     }
 
+    func testSetAndClearSubscriberAttributesPassesThroughToAttributesManager() {
```

Comment:
nitpick: I'd split these into separate cases so that if the test fails we immediately know which method was messed up

Reply:
✅